app/components/ui/Section/CategorySection.tsx

```diff
@@ -21,51 +22,152 @@ interface FeaturedCategoriesSection {
 export interface FeaturedCategoriesData {
   header: string;
   subheader: string;
-  backgroundColor: string;
+  backgroundColor: BackgroundColorVariant;
   featuredCategoriesSection: FeaturedCategoriesSection[];
 }
 
-interface CategorySectionProps {
-  sectionData: FeaturedCategoriesData;
-}
-
-export const CategorySection = (props: CategorySectionProps) => {
-  const { sectionData } = props;
+export const CategorySection = () => {
+  // const { sectionData } = props;
 
-  if (!sectionData) {
-    return <div>Loading...</div>;
-  }
+  // if (!sectionData) {
+  //   return <div>Loading...</div>;
+  // }
 
-  const { header, subheader, backgroundColor, featuredCategoriesSection } =
-    sectionData;
-  const featuredCategories = featuredCategoriesSection[0].category;
+  // const { header, subheader, backgroundColor, featuredCategoriesSection } =
+  //   sectionData;
+  // const featuredCategories = featuredCategoriesSection[0].category;
 
   return (
     <HomePageSection
-      title={header}
-      description={subheader}
-      backgroundColor={backgroundColor}
+      title={tempCategoriesSection.header}
+      description={tempCategoriesSection.subheader}
+      backgroundColor="secondary"
     >
       <div className={styles.categorySection}>
         <div className={styles.categoryCards}>
-          {featuredCategories.map((category) => {
+          {hardcodeCategories.map((category) => {
             return (
               <CategoryCard
-                key={category.label}
-                label={category.label}
-                slug={category.slug}
+                key={category.name}
+                label={category.name}
+                href={`/${category.categorySlug}/results`}
                 icon={category.icon}
               />
             );
           })}
           <CategoryCard
             key="See all services"
             label="See all services"
-            slug=""
+            href="/search"
             icon={{ provider: "fas", name: "fa-arrow-right" }}
           />
         </div>
       </div>
     </HomePageSection>
   );
 };
+
+export const hardcodeCategories = [
```

Comment:
What do you think about naming `topLevelCategories` since that's how we refer to them? Also do y'all ever use `CONST_CASE` for configuration data like this?

Reply:
Oh, good question! I added a comment to explain that it's temporary so maybe not necessary in this case but def will keep that in mind in the future for unchanging vars / data